### PR TITLE
Restaurant website added

### PR DESCRIPTION
**Pull request by Jyoti040**
Restaurant website added

### CODE REVIEW

File: Projects/Restaurant/index.html [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
...9dVs56REGww/l/alC2/YfSjHVf1H60ztng/vRsFA7BusfSmrgPb7US4PGPrTQmiARpr7IcCR1FWXwrxa5b81m4yfAn7jg1W2U+tZZuMh3A/EdDWa9BTrs6n4Z+JWrtQLgS6PmrfUY+1Wnw38WrDQLiXEPwDD7Z+1cc0+qDjHPatuKCySXYXig+j0Povb7RXIi+gPZjtP0aKndP4tafKurfAg15cXsawqVypI9QSD9RTLIvKEeD0z1YLyHtW4U15RTxrW2s29Vfjt7xzHyJNG6f2+8SXjVufiEP/AE0/KInxSPUJsqela/hk7CvNtv8AE/xMf+cp+Ntf6U8PxX8T/Xa//X/9q3JewfGz0Z/Cp2FbGnTsK86H8V/Ezw9r5W//ALU2fxM8Ub/z1Hwtr/WtzXs3xM9Ii0o6ClSo7V5iu+3fiR51dz4AIv3C0Hf8c1l38+pvn/8AI4H0BpXNDLCz1Jf8QtIJd0UepA/eoPXe3/h1n82qtk9kO8/RJrzU1ovliWPcmT96ctWTMdKV5EUWD9nbvEPxj0i/8G1du/IIv/yM/aq5r/xZ1dz/AIVu3aB65dh9YH2rnRSBk/31ovwjw+9fbbZWR+oyF+vUVN5GyqwxRJeI+O6rUH/Gv3HHbdCT/wAogfak6HQXL3/DUn/Nwsevf4CrR4X7FqsG6d7dug9AvX5zVnXSoigAADioynZVJLSKn4J7Lp+a9525Kn8n061btNpgo6DsOKTbgcYxz9qd5+OKS7C2Oo2MD5n/AEpJf9hx9K0inFENbj0j/uKIgNtnp0raLAj5UzrddbtKXdgoHUmB96p/iHtsTK6e0zH9T+VfiByftTJNmbSLqyA1qudN4nqmyXcE/pZAPkCKym4MXminkZ+dF6ZBtOBwa3WVYiMhRHFC3eaysrBB2pLDBrdZTABWrTARWVlEAnSmLgjvU0vX4/1rKykmPAUvNLXj++9ZWUhQYuc1HOPOfjWqymQrFsKQ1ZWUQGWuTRZ6fD/WsrKDCh+0o7dqXZOa3WUjHRu9gGO4/ejAPLWVlKx0b8HQNqLYYAiGMHIkHFdY8DtgWxAAmeg7msrKXJ4DHokU5pu9/wBX/VWqypvoCGrPX4Gl2+B8RWVlFGYdbH9f3oXVMZPz/asrKIqOXe0F1m1BDEkAtAJJAz0B4pKOY5PSsrK6V0iL/wBBBY9zWVlZQMf/2Q=="
```
Comment (VidhiBhatt01) [inline Projects/Restaurant/index.html:106]: why is the url so long?

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
1. Link issue in PR description
2. Attach image of how it looks in PR description

File: Projects/Restaurant/index.html (hunk 1 above)
Comment (Jyoti040) [reply Projects/Restaurant/index.html:106]: This url is of image used in website, the image was one of the stock images only.

**Comment by Jyoti040** [comment]
1. Fixes #118 
 2. 
IMG_20231029_111408

**Review by VidhiBhatt01 — APPROVED** [review]

**Pull request merged**